python/test/t_Waarts_concave.py

```diff
@@ -41,24 +41,6 @@
 
 #
 # Calculation
-# ot.
-#
-# FORM/SORM Cobyla
-myCobyla = ot.Cobyla()
```

Comment:
I guess that this block of unit test fails now that you check that the threshold is accurately reached. Am I correct? If yes, this means that Cobyla does not perform well. In this case, it might be relevant to create an issue that point this particular problem. This will prevent from forgetting this issue.

Reply:
Indeed, in this case (and other ones), Cobyla is not able to reach convergence. NLopt Cobyla performs quite well. I think this is not necessarily an issue. Depending on the problem specifications, several optimization algorithms could not find a solution.